src/azul/indexer/__init__.py

```diff
@@ -55,6 +60,78 @@ class BundleFQID(SupportsLessThan):
     version: BundleVersion
 
 
+@attr.s(frozen=True, auto_attribs=True, kw_only=True)
+class Prefix:
+    prefix: str = ''
+    partition_prefix_length: int
```

Comment:
```suggestion
    partition: int
```

It's imprecise but we need the `repr` result to be shorter for all those doc tests